core/src/main/java/com/segment/analytics/kotlin/core/platform/Plugin.kt

```diff
@@ -89,7 +89,7 @@ abstract class DestinationPlugin : EventPlugin {
     override val type: Plugin.Type = Plugin.Type.Destination
     private val timeline: Timeline = Timeline()
     override lateinit var analytics: Analytics
-    internal var enabled = false
+    internal var enabled = true
```

Comment:
👍 good catch! otherwise we might have issue when the user first opens the app offline and plugins are disabled by default.

Reply:
just found out swift actually has defaulted to false. would that cause a behavior difference?